endpoints-versions/src/main/scala/com/thenewmotion/ocpi/VersionsRoute.scala

```diff
@@ -70,9 +70,11 @@ class VersionsRoute(versions: => Map[VersionNumber, OcpiVersionConfig]) extends
 
   val VersionMatcher = Segment.flatMap(s => VersionNumber.opt(s))
 
-  def route(apiUser: GlobalPartyId)(implicit ec: ExecutionContext, mat: ActorMaterializer) = {
+
+  def route(apiUser: GlobalPartyId, securedConnection: Boolean = true)(implicit ec: ExecutionContext, mat: ActorMaterializer) = {
```

Comment:
I was skipping them before already. it's the default behaviour for our apps. in a previous commit, I enabled them hoping that this would give us "https" in the extracted URI below. But it didn't, and it refuses plain http requests which won't work with our setup, IIUC